Commands/Workflows/GetWorkflowInstances.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using SharePointPnP.PowerShell.CmdletHelpAttributes;
+using SharePointPnP.PowerShell.Commands.Base.PipeBinds;
+using Microsoft.SharePoint.Client.WorkflowServices;
+
+namespace SharePointPnP.PowerShell.Commands.Workflows
+{
+    [Cmdlet(VerbsCommon.Get, "PnPWorkflowInstance")]
+    [CmdletHelp("Get workflow instances",
+        "Gets all workflow instances",
+        Category = CmdletHelpCategory.Workflows)]
+    [CmdletExample(
+        Code = @"Get-PnPWorkflowInstance -Item $SPListItem",
+        Remarks = "Retreives workflow instances running against the provided item",
+        SortOrder = 1)]
+    [CmdletExample(
+        Code = @"Get-PnPWorkflowInstance -Item $SPListItem",
+        Remarks = "Retreives workflow instances running against the provided item",
+        SortOrder = 2)]
+
+    public class GetWorkflowInstance : PnPWebCmdlet
+    {
+        [Parameter(Mandatory = true, HelpMessage = "The SPList for which workflow instances should be retreived", Position = 0)]
```

Comment:
Use a ListPipeBind object instead and do not refer to 'development' objects like SPList. A ListPipeBind type of parameter will take a list object, an idea or the name of a the list. See the Get-PnPList cmdlet for instance for an example.